projects/canopy/src/lib/header/account-menu/account-menu.component.scss

```diff
@@ -14,13 +14,16 @@
 
 a.lg-account-menu-item,
 button.lg-account-menu-item {
-  display: flex;
-  flex-direction: column;
-  padding: var(--space-xxs);
-  position: relative;
-
   @include mixins.lg-nav-item();
 
+  /* stylelint-disable no-duplicate-selectors */
```

Comment:
Could you explain why you have to add `/* stylelint-disable no-duplicate-selectors */`?

Is it because that selector is already in the mixins? or is it complaining about the &?